ceremonies/src/lib.rs

```diff
@@ -164,10 +185,38 @@ decl_module! {
                 // register participant as reputable
                 <ParticipantReputation<T>>::insert((cid, cindex),
                     &sender, Reputation::UnverifiedReputable);
+
+                participant_has_reputation = true;
             };
-            <ParticipantRegistry<T>>::insert((cid, cindex), &new_count, &sender);
-            <ParticipantIndex<T>>::insert((cid, cindex), &sender, &new_count);
-            <ParticipantCount>::insert((cid, cindex), new_count);
+
+            if <encointer_communities::Module<T>>::bootstrappers(cid)
+                .contains(&sender) {
+                let participant_index = <BootstrapperCount>::get((cid, cindex)).checked_add(1).
+                    ok_or("[EncointerCeremonies]: Overflow adding reputable to registry")?;
+                <BootstrapperRegistry<T>>::insert((cid, cindex), &participant_index, &sender);
+                <BootstrapperIndex<T>>::insert((cid, cindex), &sender, &participant_index);
+                <BootstrapperCount>::insert((cid, cindex), participant_index);
```

Comment:
these statements are very alike for every if-block.. abstracting them into a function is not meaningful, because the registry, index and count do not have any explicit relation in the code.. would they have been "hierarchical" in a way of having a BootstrapperData-object, a ReputableData-object which each contain the three maps etc., one could have made a function by passing said data object and inserting the arguments accordingly.. (this is more fyi)
in here, I would propose to at least add a comment that states that all if-blocks do basically the same.. this saves time for every developer who has to read the code.. also the entirety of the if-else-if-else-block could be moved to a function...